src/llfs/volume.hpp

```diff
@@ -197,6 +197,11 @@ class Volume
   //
   const PageRecycler::Metrics& page_recycler_metrics() const;
 
+  LogDevice& root_log() const noexcept
+  {
+    return *this->root_log_;
```

Comment:
Not sure if 
return *(this->root_log_);
will improve readability.

Reply:
I can change it, if you think it helps!